content/SCALE/GettingStarted/FirstTimeLogin.md

```diff
@@ -104,25 +104,25 @@ The Task Manager displays all running and failed jobs/processes.
 
 ![TaskManagerSCALE](/images/SCALE/TaskManagerSCALE.png "TrueNAS SCALE Task Manager")
 
-Users can click the *History* button to open the **Jobs** screen. **Jobs** lists all *Successful*, *Active*, and *Failed* jobs. Users can also click *View Logs* next to a failed process to view its log information and error message.
+Users can click the **History** button to open the **Jobs** screen. **Jobs** lists all Successful, Active, and Failed jobs. Users can also click **View Logs** next to a failed process to view its log information and error message.
```

Comment:
If specifically talking about a UI element ,such as a table column header use bold and capitalize, but if talking generally about a class/category lower case and no bold is proper.
```suggestion
Users can click the **History** button to open the **Jobs** screen. **Jobs** lists all successful, active, and failed jobs. Users can also click **View Logs** next to a failed process to view its log information and error message.
```